test/built-ins/RegExp/prototype/dotAll/this-val-invalid-obj.js

```diff
@@ -16,6 +16,9 @@ features: [regexp-dotall]
 ---*/
 
 var dotAll = Object.getOwnPropertyDescriptor(RegExp.prototype, 'dotAll').get;
+var other = $262.createRealm().global;
+var otherRegExpProto = other.RegExp.prototype;
+var otherRegExpGetter = Object.getOwnPropertyDescriptor(otherRegExpProto, 'dotAll').get;
```

Comment:
Please put all of the newly added cross-realm test code into a new test file. Here are helpful examples to follow: https://github.com/tc39/test262/search?p=1&q=cross-realm&unscoped_q=cross-realm


(Apply this same review feedback to all files in this PR)

Reply:
Done.